Dockerfile

```diff
@@ -1,7 +1,7 @@
-FROM alpine:3.13
+FROM alpine:3.17.2
 
 # This makes it easy to build tagged images with different `kubectl` versions.
-ARG KUBECTL_VERSION="v1.13.12"
+ARG KUBECTL_VERSION="v1.26.1"
```

Comment:
Regarding changing the `KUBECTL_VERSION`.
Would You be so kind and update the version here:
https://github.com/criblio/k8s-webhook-cert-manager/blob/0f4255fbcae051f953e0da22c626275794c7e432/Makefile#L3
in a Makfile

Reply:
done.